src/engine/sim_engine/blockdev.rs

```diff
@@ -37,8 +37,13 @@ impl BlockDev for SimDev {
         self.user_info.as_ref().map(|x| &**x)
     }
 
-    fn set_user_info(&mut self, user_info: Option<&str>) -> () {
-        self.user_info = user_info.map(|x| x.to_owned());
+    fn set_user_info(&mut self, user_info: Option<&str>) -> bool {
```

Comment:
Might as well macroize this and its twin.

Reply:
I'd rather not, this is less than the amount of code I think warrants a macro.

Reply:
It makes sense to make things that are shared by both the engines into macros, regardless of size. That way we know that they are shared. That is the main point of `src/engine/macros.rs`.

Reply:
ok that makes sense to me, will do.